irs/src/App.js

```diff
@@ -6,14 +6,17 @@ import Login from './components/Login/Login';
 import Register from './components/Registration/Register';
 import Whoyouare from './components/Whoyouare/Whoyouare';
 import MainNav from './components/Navigation/MainNav';
+import DashboardPage from './components/Dashboard/Dashboard';
 
 class App extends Component {
   render() {
     return (
       <Router>
         <div>
           <MainNav />
-          <Route exact path="/" />
+          <switch>
+            <Route exact path="/" component={DashboardPage} />
+          </switch>
```

Comment:
you need to be importing `Switch` from react-router-dom here for this to work, but why the switch statement just around the "/" path? You already have `exact`, but if you want to only display one of all the different routers at any given time they would all need to be wrapped in `Switch`